tests/test_todo.py

```diff
@@ -72,32 +72,11 @@ def test_duration() -> None:
     ("params"),
     [
         ({}),
-        (
-            {
-                "start": datetime.datetime(2022, 9, 6, 6, 0, 0),
-            }
-        ),
         (
             {
                 "due": datetime.datetime(2022, 9, 6, 6, 0, 0),
             }
         ),
-        (
-            {
```

Comment:
Can we add tests for how these cases do behave w.r.t. recurring TODO? You mentioned duration having an effect here, though not sure that happens as of this PR yet. Any other tests for duration that need to be added or is existing coverage good?

Thank you for your contribution.

Reply:
I see you had a question about this in the issue but i didn't understand the question exactly.